sdk/storage/storage-blob/test/appendblobclient.spec.ts

```diff
@@ -63,4 +65,140 @@ describe("AppendBlobClient", () => {
     assert.equal(await bodyToString(downloadResponse, content.length), content);
     assert.equal(downloadResponse.contentLength!, content.length);
   });
+
+  it("can be created with a url and a credential", async () => {
+    const factories = appendBlobClient.pipeline.factories;
+    const credential = factories[factories.length - 1] as SharedKeyCredential;
+    const newClient = new AppendBlobClient(appendBlobClient.url, credential);
+
+    await newClient.create();
+    await newClient.download();
+  });
+
+  it("can be created with a url and a credential and an option bag", async () => {
+    const factories = appendBlobClient.pipeline.factories;
+    const credential = factories[factories.length - 1] as SharedKeyCredential;
+    const newClient = new AppendBlobClient(appendBlobClient.url, credential, {
+      telemetry: { value: "test/1.0" }
+    });
+
+    await newClient.create();
+    await newClient.download();
+  });
+
+  it("can be created with a url and a pipeline", async () => {
+    const factories = appendBlobClient.pipeline.factories;
+    const credential = factories[factories.length - 1] as SharedKeyCredential;
+    const pipeline = newPipeline(credential);
+    const newClient = new AppendBlobClient(appendBlobClient.url, pipeline);
+
+    await newClient.create();
+    await newClient.download();
+  });
+
+  it("can be created with a connection string", async () => {
+    const newClient = new AppendBlobClient(
+      process.env.CONNECTION_STRING || "",
```

Comment:
How about adding a helper function `getConnectionStringForTest()` or something, in `test/utils/index.ts` to return the connection string?  There are already functions there to get credentials so it feels a proper place to have all the logic there. Also report error if connection string is not set in env variable, similar to what's done for account name/key.